nixos/doc/manual/release-notes/rl-2009.xml

```diff
@@ -400,6 +400,22 @@ systemd.services.nginx.serviceConfig.ReadWritePaths = [ "/var/www" ];
       The rkt module has been removed, it was archived by upstream.
     </para>
    </listitem>
+   <listitem>
+     <para>
+       In addition to the hostname, the fully qualified domain name (FQDN),
+       which consists of <literal>${cfg.hostName}</literal> and
+       <literal>${cfg.domain}</literal> is now added to
+       <literal>/etc/hosts</literal>, to fix the
+       <literal>hostname --fqdn</literal> command and other applications that
+       try to determine the FQDN. These new entries take precedence over entries
+       from the DNS which could cause regressions in some very specific setups.
+       Additionally the hostname is now resolved to <literal>127.0.0.2</literal>
+       instead of <literal>127.0.1.1</literal> to match the behaviour of
+       <literal>nss-myhostname</literal> (systemd).
```

Comment:
```suggestion
       instead of <literal>127.0.1.1</literal> to be consistent with what
       <literal>nss-myhostname</literal> (from systemd) returns.
```